axelrod/strategies/finite_state_machines.py

```diff
@@ -55,16 +81,16 @@ class FSMPlayer(Player):
         'manipulates_state': False
     }
 
-    def __init__(self, transitions=None, initial_state=None,
-                 initial_action=None) -> None:
+    def __init__(self, transitions: List[Tuple[int, Any, int, Any]] =None, initial_state: int =None,
```

Comment:
do you mean `def something(num: int =5):` should be `def something(num: int = 5)`?  if that's it, i wasn't sure of the spacing convention for type hints.